mod_ci/controllers.py

```diff
@@ -189,6 +189,21 @@ def delete_expired_instances(compute, max_runtime, project, zone) -> None:
             creationTimestamp = datetime.datetime.strptime(instance['creationTimestamp'], '%Y-%m-%dT%H:%M:%S.%f%z')
             currentTimestamp = datetime.datetime.now(datetime.timezone.utc)
             if currentTimestamp - creationTimestamp >= datetime.timedelta(minutes=max_runtime):
+                # Update test status in database and on GitHub
+                platform_name, test_id = vm_name.split('-')
+                test = Test.query.filter(Test.id == test_id).first()
+                message = "Could not complete test, time limit exceeded"
```

Comment:
Should we add this string as a constant and then use it here?

We can add the constant with other such messaging constants file.

Reply:
Actually we should not do this, at least now because:
1. This would have made sense if we could attach a description to the status itself, but since this would somewhat restrict custom messages for the same status, we should allow free text
2. We are not reusing 2 same messages, so even duplication isn't an issue

But definitely if in future it is required, we can make this change...

Reply:
Actually, we should not do this, at least now, because:

1. This would have made sense if we could attach a description to the status itself, but since this would somewhat restrict custom messages for the same status, we should allow free text
2. We are not reusing two same messages, so even duplication isn't an issue

But definitely a good suggestion. If in future it is required, we can make this change... Thoughts?